lambda/rotator-snowflake/main.go

```diff
@@ -31,30 +31,32 @@ func Rotate(ctx context.Context) error {
 
 	snowflakeAccounts, err := setup.Snowflake(ctx, secretStore)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "Unable to configure snowflake accounts")
 	}
 
 	databricksUsers, err := setup.ListDatabricksUsers(ctx, oktaClient, databricksAccount)
 	if err != nil {
 		return errors.Wrap(err, "Unable to get list of users to rotate")
 	}
 
-	accountErrors := &multierror.Error{}
+	userErrors := &multierror.Error{}
 	for _, snowflakeAccount := range snowflakeAccounts {
 		snowflakeUsers, err := setup.ListSnowflakeUsers(ctx, oktaClient, snowflakeAccount)
 		if err != nil {
-			accountErrors = multierror.Append(accountErrors, errors.Wrap(err, "Unable to list Snowflake Users"))
+			return errors.Wrap(err, "Unable to list Snowflake Users")
+		}
+		if snowflakeUsers == nil {
+			return errors.Errorf("Got no snowflakeUsers listed for %s account", snowflakeAccount.Name)
```

Comment:
The biggest change here: if one Snowflake account fails to get users, then return an error. Don't skip it.